apps/journeys-admin/src/components/JourneyQuickSettings/JourneyQuickSettings.spec.tsx

```diff
@@ -0,0 +1,72 @@
+//
+
+import { MockedProvider } from '@apollo/client/testing'
+import { fireEvent, render, screen } from '@testing-library/react'
+
+import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
+
+import { journey } from '../../libs/useHostCreate/useHostCreate.mocks'
+
+import { JourneyQuickSettings } from './JourneyQuickSettings'
+
+jest.mock('@mui/material/useMediaQuery', () => ({
+  __esModule: true,
+  default: jest.fn()
+}))
+
+describe('JourneyQuickSettings', () => {
+  it('should render elements', () => {
+    render(
+      <MockedProvider>
+        <JourneyProvider value={{ journey, variant: 'admin' }}>
+          <JourneyQuickSettings />
+        </JourneyProvider>
+      </MockedProvider>
+    )
+
+    expect(screen.getByTestId('OnboardingTemplateCard')).toBeInTheDocument()
+    expect(screen.getByTestId('JourneyQuickSettingsTabs')).toBeInTheDocument()
+    expect(screen.getByTestId('JourneyQuickSettingsChat')).toBeInTheDocument()
+    expect(
+      screen.queryByTestId('JourneyQuickSettingsGoals')
+    ).not.toBeInTheDocument()
+    expect(screen.getByTestId('PreviewItem')).toBeInTheDocument()
+    expect(screen.getByRole('button', { name: 'Share' })).toBeInTheDocument()
+  })
+
+  it('should render journeyquicksettingsgoals on goals tab click', () => {
```

Comment:
```suggestion
  it('should render journeyQuickSettingsGoals on goals tab click', () => {
```
Any particular reason for this not being in camelCase? We have words with no spaces in camelCase for readability.